driver/src/util.rs

```diff
@@ -104,9 +106,23 @@ pub fn batch_processing_state(
 
 pub fn create_price_finder(fee: Option<Fee>) -> Box<dyn PriceFinding> {
     let solver_env_var = env::var("LINEAR_OPTIMIZATION_SOLVER").unwrap_or_default();
+    let optimization_model_string: String =
```

Comment:
How about having one variable here and using naive, mip, and nlp as flags, rather than two variables (e.g. the combination LINEAR_OPTIMIZATION_SOLVER=0, OPTIMIZATION_MODEL=mip seems invalid.